web_edit_user_filter/static/src/xml/backend.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<template>
+    <div t-name="web_edit_user_filter.Popover">
+
+        <div class="list-group">
+            <t t-foreach="values" t-as="value">
+                <a class="list-group-item list-group-item-action"
+                   t-attf-data-value="#{typeof value.value === 'object' ? value.value.attrs.name : value.value}">
+                    <span>
+                        <t t-esc="value.label"/>
+                    </span>
+                    <button class="btn btn-sm btn-link pull-right">
+                        <span class="fa fa-close" />
+                    </button>
+                </a>
+            </t>
+        </div>
+
+        <button type="button" class="btn btn-primary btn-block mt-2 btn-done">
+            Done
```

Comment:
It seems to me like this button is not needed. Just clicking outside the bubble should close it.

Reply:
I removed it and made a simple mechanism to hide the popovers.